test/cbmc/.gitignore

```diff
@@ -0,0 +1,22 @@
+# Emitted when running CBMC proofs.
+proofs/**/logs
+proofs/**/gotos
+proofs/**/report
+proofs/**/html
+
+# Emitted by CBMC Viewer.
+TAGS-*
+
+# Emitted by Arpa
+arpa_cmake/
+arpa-validation-logs/
+Makefile.arpa
+
+# These files should be overwritten whenever prepare.py runs.
+cbmc-batch.yaml
```

Comment:
No need for this line I think. We're not generating or committing `cbmc-batch.yaml` any more.

Reply:
But we do need some touch file like cbmc-batch.yaml in the proof directories to mark the directory as containing a proof.  I don't see any in this pull request.